internal/controller/operatorconfigmap_controller.go

```diff
@@ -882,6 +907,21 @@ func (c *OperatorConfigMapReconciler) reconcileClientOperatorSubscription() erro
 	return nil
 }
 
+func (c *OperatorConfigMapReconciler) reconcileNoobaaOperatorSubscription() error {
+	noobaaSubscription, err := c.getSubscriptionByPackageName("noobaa-operator")
+	if err != nil {
+		return err
+	}
+	desiredSubChannel := utils.GetNoobaaOperatorChannel()
+	if desiredSubChannel != "" && desiredSubChannel != noobaaSubscription.Spec.Channel {
```

Comment:
Looking at the impl of `GetNoobaaOperatorChannel`, the function cannon return "". So why are we testing for that?

Reply:
you are correct, changed.